client-spark/spark2-shaded/pom.xml

```diff
@@ -216,6 +216,10 @@
                   <pattern>picocli</pattern>
                   <shadedPattern>${rss.shade.packageName}.picocli</shadedPattern>
                 </relocation>
+                <relocation>
+                  <pattern>com.codahale.metrics</pattern>
+                  <shadedPattern>${rss.shade.packageName}.com.codahale.metrics</shadedPattern>
```

Comment:
check the dependencies of this is shaded too

Reply:
<img width="1462" alt="image" src="https://github.com/user-attachments/assets/dfd6190e-2a1f-4f4c-b81e-57c26df01551">